src/components/Nav.js

```diff
@@ -18,14 +20,54 @@ const Nav = () => {
       alignSelf="center"
       gap="medium"
     >
-      <RoutedAnchor
-        path="/"
-        icon={<GrommetIcon size="large" />}
-        label={size !== 'small' && <Text size="xlarge">grommet</Text>}
-      />
+      <Box
+        direction="column"
+        justify="between"
+        align="start"
+        alignSelf="center"
+        width="medium"
+      >
+        <RoutedAnchor
+          path="/"
+          icon={<GrommetIcon size="large" />}
+          label={size !== 'small' && <Text size="xlarge">grommet</Text>}
+        />
+        {themeName === 'grommet' && <Text size="small">for open source</Text>}
+        {themeName === 'hpe' && <Text size="small">for HPE</Text>}
+      </Box>
+
       <Box direction="row" gap="small">
         {!searchOpen && (
           <Box direction="row" gap="small">
+            <Button
+              plain
```

Comment:
When button has children, I think it automatically behaves as `plain`, so we could remove this.

Reply:
![image](https://user-images.githubusercontent.com/23144209/179321118-31305f6a-28fc-4501-9e4a-c8d809eef19e.png)
When I removed `plain`, this happened. Not sure if there is another fix around it, but would keeping `plain` be the most effective solution?

Reply:
oh interesting, okay then leave `plain` !